### DISCUSSION TITLE
AniList connection seems to be broken now, API responds with bad request every time I try to authenticate

### DISCUSSION THREAD

**Discussion by tzheliang** [vosmiic/jellyfin-ani-sync#142] [Q&A]
Enclosed the log I received when I tried to authenticate using `Test Authentication` for AniList. It's been a while since I last used this plugin but it was working fine before and not working properly now.

`[23:12:35] [ERR] [23] jellyfin_ani_sync.Api.AuthApiCall: Unable to complete AniList API call (POST https://graphql.anilist.co), reason: BadRequest; Bad Request`

**Comment by vosmiic** [comment]
What version of Jellyfin and the plugin are you using? Authentication for AniList seems to be working on my side. I think the logging needs improving; only returning the status code and reason phrase obviously does not tell us the reason for the bad request.

**Reply by tzheliang** [reply]
Jellyfin: V10.9.6
Ani-Sync: V3.2.0.0

Agreed on the logging, though were there any major changes in the plugin setup in the last few months? Perhaps my configuration is out of date

**Reply by vosmiic** [reply]
>Agreed on the logging, though were there any major changes in the plugin setup in the last few months? Perhaps my configuration is out of date

I don't think so; nothing that would break the authentication process at least. Have you tried going through the whole authentication process again (if its not too much effort)?
Also I have updated the unsuccessful response error logging to include the response content. Should help with debugging issues in the future.
